data/blog/2024-year-in-review.mdx

```diff
@@ -0,0 +1,545 @@
+program](/blog/announcing-lts-grant-program-to-support-bitcoin-core-contributors)
+and aim to be a low-time preference, long-term thinking organization. If you are
+in a position to make such a commitment, please don’t hesitate to [reach out to
+us](/contact).
+
+In terms of amounts received, we are happy to report that we received higher
+amounts via bitcoin directly overall, with over half a million dollars (\$
+526,044) received via our “[donate with bitcoin](/donate)” button on our
+website. This is about 4x as much as we received via fiat payment rails (\$
+133,050)—excluding the large donations mentioned above. The ratio of on-chain to
+off-chain donations stayed about the same as last year, with ~70% of bitcoin
+payments made using the Lightning Network. In terms of number of donations,
+bitcoin rails continue to dominate.
+
+![](/static/images/blog/2024-eoy/donations.png)
+
+The vast majority of donations (~80%) went to our [General
+Fund](/funds/general), with the remainder split about evenly between [The Nostr
+Fund](/funds/nostr) and our [Operations Budget](/funds/ops). In terms of
+amounts, around ~95% of money donated by the general public is directed towards
+our General Fund.
+
+<center>
+    <DonateToGeneralFundButton/>
+</center>
+
+---
+
+## Allocation
+
+The purpose of our two funds is to support free and open-source projects that
+are aligned with [our mission](/mission). The [Nostr Fund](/funds/nostr) is
+dedicated towards nostr projects, a protocol that is just coming out of its
+infancy. The [General Fund](/funds/general) is dedicated towards all other free
+and open-source projects and contributors, especially bitcoin-related projects
+and projects that help bitcoin flourish.
+
+As of December 2024, we have allocated over \$12M in grants from the General
+Fund, and close to \$9M in grants from The Nostr Fund. In terms of money
+allocated [year-over-year](/blog/2023-year-in-review#allocation) this is an
+increase of 67% on the nostr side, and 81% on the bitcoin side. When looking at
+total allocation in USD terms, our two funds are at ~89% and ~52% allocated,
+respectively.
+
+![](/static/images/blog/2024-eoy/allocation.png)
+
+As mentioned in [last year’s report](/blog/2023-year-in-review), we suspect that
+the distinction between “bitcoin” and “nostr” will get more blurry over time, as
+nostr matures and gets integrated into a variety of bitcoin software across the
+stack. This is already happening, especially on the eCash front. Are [Cashu
+addresses](/blog/nostr-grants-july-2024#npubcash) a Lightning thing, a bitcoin
+thing, or a nostr thing? Are wallets that use nostr for social payments nostr
+clients, or bitcoin wallets? And what about P2P exchanges that use nostr to
+publish their order books? It is obvious to us that various concepts and
+technologies are currently merging, and we expect this cross-pollination of
+ideas to continue.
+
+With all that said, nostr is still an incredibly young network. It is just
```

Comment:
"It is just coming out of its infancy" is repetitive. Just say "incredibly young network, which is why..."